packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -24,17 +26,54 @@ const RowComponent = React.memo(({ data: advert, showAdPopup, style }) => {
-                </Table.Cell>
-                <Table.Cell className='advertiser-page__adverts-price'>
-                    {price_display} {local_currency}
-                </Table.Cell>
+                {isMobile() ? (
+                    <React.Fragment>
+                        <Table.Cell className='advertiser-page__cell'>
+                            <Text size='xxs' lineHeight='m'>
+                                <Localize
+                                    i18n_default_text='Rate (1 {{currency}})'
+                                    values={{
+                                        currency,
+                                    }}
+                                />
+                            </Text>
+
+                            <div className='advertiser-page__adverts-price'>
+                                <Text color='profit-success' size='s' weight='bold' lineHeight='m'>
+                                    {price_display} {local_currency}
+                                </Text>
+                            </div>
+                            <div className='advertiser-page__cell-limit'>
+                                <Text size='xxs' lineHeight='m'>
+                                    <Localize
+                                        i18n_default_text='Limit {{min_order_amount_limit_display}}-{{max_order_amount_limit_display}} {{currency}}'
+                                        values={{
+                                            min_order_amount_limit_display,
+                                            max_order_amount_limit_display,
+                                            currency,
+                                        }}
+                                    />
+                                </Text>
+                            </div>
+                        </Table.Cell>
+                    </React.Fragment>
+                ) : (
+                    <React.Fragment>
+                        <Table.Cell>
+                            {`${min_order_amount_limit_display}-${max_order_amount_limit_display} ${currency}`}
+                        </Table.Cell>
+                        <Table.Cell className='advertiser-page__adverts-price'>
+                            <Text color='profit-success' size='xs' weight='bold' lineHeight='xs'>
+                                {price_display} {local_currency}
+                            </Text>
+                        </Table.Cell>
+                    </React.Fragment>
+                )}
                 {is_my_advert ? (
                     <Table.Cell />
                 ) : (
                     <Table.Cell className='advertiser-page__adverts-button'>
-                        <Button primary small onClick={() => showAdPopup(advert)}>
+                        <Button primary small={!isMobile()} large={isMobile()} onClick={() => showAdPopup(advert)}>
```

Comment:
```suggestion
                        <Button primary {...(isMobile() ? {large: true} : {small: true})} onClick={() => showAdPopup(advert)}>
```

Reply:
Thank you for you review, Salar! 🙏 
I have incorporated some of your suggested changes.